frontend-react/src/pages/home/Sections/Section.tsx

```diff
@@ -1,12 +1,9 @@
 import { SectionProp } from "../HomeProps";
 
-import CtaSection from "./CtaSection";
 import LiveMapSection from "./LiveMapSection";
 
 export default function Section({ section }: { section: SectionProp }) {
-    if (section.type === "cta") return <CtaSection section={section} />;
-    else if (section.type === "liveMap")
-        return <LiveMapSection section={section} />;
+    if (section.type === "liveMap") return <LiveMapSection section={section} />;
```

Comment:
Should we add the section types to an enum?

Reply:
Negative. The Home page is expected to go through a complete rewrite in the future.